### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.text)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2292]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
.approve.red.labeled > .text
#### Snippet
`<span class="ui text">Reset</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.78 (foreground color: #ffffff, background color: #ef767a, font size: 12.0pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

reset button in reset dialogue

![Screen Shot 2019-07-18 at 2 38 22 PM](https://user-images.githubusercontent.com/5615930/61493946-b7b0fa80-a969-11e9-90e5-9ea62d00e4e1.png)

**Comment by jwunderl** [comment]
This will be fixed by https://github.com/microsoft/pxt-microbit/pull/2457, another commit that used to contain the change closed this a bit too early

**Comment by BhuwanNaswa** [comment]
Labels
#A11yMAS, #HCL, #HCL-MakeCode-E2E, #MAS1.4.3, #Win10-Chrome, #Win10-Edge, #accessibility, #K4W, #closed
